### PR TITLE
[WIP] [ENT-98] Bulk Unlink or deactivate enterprise users

### PR DESCRIPTION
**Pull request by replaceafill**
**Description:** This PR is not complete yet. More info to follow.

**JIRA:** [ENT-98](https://openedx.atlassian.net/browse/ENT-98)

**Dependencies:** None 

**Installation instructions:** Normal installation.

**Testing instructions:**

TBD

**Reviewers:**
- [ ] @haikuginger  

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are (reasonably) squashed
- [ ] Translations are updated
- [ ] PR author is listed in AUTHORS

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPi after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @replaceafill! I've created [OSPR-1676](https://openedx.atlassian.net/browse/OSPR-1676) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/edx-enterprise/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: enterprise/models.py [hunk 1]
```diff
@@ -167,6 +167,12 @@ class EnterpriseCustomerUserManager(models.Manager):
     This class should contain methods that create, modify or query :class:`.EnterpriseCustomerUser` entities.
     """
 
+    def get_queryset(self):
+        """
+        Return a new QuerySet object. Filters out inactive Enterprise Customer Users.
+        """
+        return super(EnterpriseCustomerUserManager, self).get_queryset().filter(active=True)
```
Comment (haikuginger) [inline enterprise/models.py]: Clever! I like this implementation a lot.

File: enterprise/models.py [hunk 2]
```diff
@@ -242,8 +263,12 @@ class EnterpriseCustomerUser(TimeStampedModel):
     )
     user_id = models.PositiveIntegerField(null=False, blank=False)
 
+    active = models.BooleanField(default=True)
```
Comment (haikuginger) [inline enterprise/models.py]: I think it's the right move to include this only on the EnterpriseCustomerUser and not the PendingEnterpriseCustomerUser. For the pECU, we don't really have enough data to make it worthwhile to keep the record around - the user has never actually interacted with the system.

File: enterprise/templates/enterprise/admin/manage_learners.html [hunk 3]
```diff
@@ -138,19 +141,41 @@
...
   </div>
   <div class="forms-panel">
-    <h1>{% trans "Link learners" %}</h1>
     <form action="" method="post" enctype="multipart/form-data" id="learner-management-form">
+      <h1>{% trans "Link learners" %}</h1>
       {% csrf_token %}
       {# as_p will render the form fields wrapped in <p> tags: #}
       {{ manage_learners_form.as_p }}
       <input type="submit" value="Submit"/>
+      <h1>{% trans "Unlink learners" %}</h1>
```
Comment (haikuginger) [inline enterprise/templates/enterprise/admin/manage_learners.html]: The UI could use some work. Function before form and all that, but it should be more clearly separated from the rest of the form.

**Comment by mattdrayer** [comment]
@bradenmacdonald @haikuginger -- what do you want to do about this PR?

**Comment by bradenmacdonald** [comment]
Closing this PR - we will revisit if/when business needs dictate.

**Pull request closed without merging**